Maxxor.PCL/Result/MxResult.cs

```diff
@@ -56,7 +58,18 @@ public static MxResult<T> Ok<T>(T value)
         }
 
         #endregion
-        
+
+        #region WithSuccessMessage
+
+        public MxResult WithSuccessMessage(string successMessage)
```

Comment:
Is this derived version needed? Can we get away with only using the one in the base?